packages/hardhat-chai-matchers/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@nomiclabs/hardhat-chai-matchers",
+  "version": "0.0.1",
+  "description": "Hardhat utils for testing",
+  "homepage": "https://github.com/nomiclabs/hardhat/tree/master/packages/hardhat-chai-matchers",
+  "repository": "github:nomiclabs/hardhat",
+  "author": "Nomic Labs LLC",
+  "license": "MIT",
+  "main": "dist/src/index.js",
+  "types": "dist/src/index.d.ts",
+  "keywords": [
+    "ethereum",
+    "smart-contracts",
+    "hardhat",
+    "testing"
+  ],
+  "scripts": {
+    "lint": "yarn prettier --check && yarn eslint",
+    "lint:fix": "yarn prettier --write && yarn eslint --fix",
+    "eslint": "eslint 'src/**/*.ts' 'test/**/*.ts'",
+    "prettier": "prettier \"**/*.{js,md,json}\"",
+    "test": "mocha --recursive \"test/**/*.ts\" --exit --reporter dot",
+    "build": "tsc --build .",
+    "clean": "rimraf dist internal types *.{d.ts,js}{,.map} build-test tsconfig.tsbuildinfo"
+  },
+  "files": [
+    "dist/src/",
+    "src/",
+    "internal/",
+    "types/",
+    "*.d.ts",
+    "*.d.ts.map",
+    "*.js",
+    "*.js.map",
+    "LICENSE",
+    "README.md"
+  ],
+  "devDependencies": {
+    "@types/bn.js": "^5.1.0",
+    "@types/chai": "^4.2.0",
+    "@types/mocha": "^9.0.0",
+    "@types/node": "^12.0.0",
+    "@typescript-eslint/eslint-plugin": "4.29.2",
+    "@typescript-eslint/parser": "4.29.2",
+    "bignumber.js": "^9.0.2",
+    "bn.js": "^5.1.0",
+    "eslint": "^7.29.0",
+    "eslint-config-prettier": "8.3.0",
+    "eslint-plugin-import": "2.24.1",
+    "eslint-plugin-prettier": "3.4.0",
+    "ethers": "^5.0.0",
+    "hardhat": "^2.0.0",
+    "mocha": "^7.2.0",
+    "prettier": "2.4.1",
+    "rimraf": "^3.0.2",
+    "ts-node": "^8.1.0",
+    "typescript": "~4.5.2"
+  },
+  "peerDependencies": {
+    "ethers": "^5.0.0",
+    "hardhat": "^2.0.0"
+  },
+  "dependencies": {
+    "chai": "^4.2.0"
```

Comment:
Shouldn't this be a peer dependency too?

Reply:
Addressed in https://github.com/NomicFoundation/hardhat/pull/2428/commits/e27c53b52117e9ae2974ce966e1954adbd00c69f

Reply:
In that https://github.com/NomicFoundation/hardhat/commit/e27c53b52117e9ae2974ce966e1954adbd00c69f I accidentally removed it as a devDependency, but I restored it in https://github.com/NomicFoundation/hardhat/pull/2428/commits/e6038e832f2111b0985a605084cf074545141faf